aiotractive/api.py

```diff
@@ -68,13 +76,27 @@ async def request(self, *args, **kwargs):
 
     async def raw_request(self, uri, params=None, data=None, method="GET"):
         """Perform request."""
+        self._attempt += 1
+
         async with self.session.request(
             method,
             self.API_URL.join(URL(uri)).update_query(params),
             json=data,
             headers=await self.auth_headers(),
             timeout=self._timeout,
         ) as response:
+            _LOGGER.debug("Request %s, status: %s", response.url, response.status)
+
+            if response.status == 429:
+                if self._attempt <= self._retry_count:
+                    delay = self._attempt**2 + random.uniform(0, 3)
```

Comment:
The delay calculation follows your guidelines. Is this enough to continue or do you want the delay counting logic to be passed through a parameter?